package.json

```diff
@@ -38,12 +38,12 @@
     "@commitlint/cli": "8.2.0",
     "@commitlint/config-conventional": "8.2.0",
     "@commitlint/travis-cli": "8.2.0",
-    "babel-eslint": "10.0.3",
+    "babel-eslint": "10.1.0",
     "babel-loader": "^8.0.4",
     "cz-conventional-changelog": "^3.0.2",
-    "eslint": "5.7.0",
-    "eslint-config-scratch": "5.1.0",
-    "eslint-plugin-react": "7.12.4",
```

Comment:
Is the addition of the major version range `^` for eslint-config-scratch and eslint-plugin-react but not eslint or babel-eslint intentional?